packages/app/docs/stories/appSpinner.story.js

```diff
@@ -60,7 +76,34 @@ export const spinner = () => {
     );
 };
 
+export const arcSpinnerWithChild = () => {
+    const customSpinner = (
+        <AppSpinner
+            type="arc"
+            size="x-large"
+            variant="brand"
+            label={<Label>loading react-rainbow</Label>}
+        >
+            <SpinnerLogo />
+        </AppSpinner>
+    );
```

Comment:
move custom spinner out of the arcSpinnerWithChild function since does not depend on any internal scope